packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -21,69 +35,363 @@ import styled from 'styled-components';
 import { ReleaseModal, FormValues } from '../components/ReleaseModal';
 import { PERMISSIONS } from '../constants';
 import { isAxiosError } from '../services/axios';
-import { useUpdateReleaseMutation } from '../services/release';
+import {
+  GetReleaseActionsQueryParams,
+  useGetReleaseActionsQuery,
+  useGetReleaseQuery,
+  useUpdateReleaseMutation,
+} from '../services/release';
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleaseDetailsLayout
+ * -----------------------------------------------------------------------------------------------*/
+const ReleaseInfoWrapper = styled(Flex)`
+  align-self: stretch;
+  border-bottom-right-radius: ${({ theme }) => theme.borderRadius};
+  border-bottom-left-radius: ${({ theme }) => theme.borderRadius};
+  border-top: 1px solid ${({ theme }) => theme.colors.neutral150};
+`;
 
 const PopoverButton = styled(Flex)`
   align-self: stretch;
 `;
 
 const PencilIcon = styled(Pencil)`
-  width: 16px;
-  height: 16px;
+  width: ${({ theme }) => theme.spaces[4]};
+  height: ${({ theme }) => theme.spaces[4]};
   path {
     fill: ${({ theme }) => theme.colors.neutral600};
   }
 `;
 
 const TrashIcon = styled(Trash)`
-  width: 16px;
-  height: 16px;
+  width: ${({ theme }) => theme.spaces[4]};
+  height: ${({ theme }) => theme.spaces[4]};
   path {
     fill: ${({ theme }) => theme.colors.danger600};
   }
 `;
 
-const ReleaseInfoWrapper = styled(Flex)`
-  align-self: stretch;
-  border-radius: 0 0 4px 4px;
-  border-top: 1px solid ${({ theme }) => theme.colors.neutral150};
-`;
+interface PopoverWrapperProps {
+  onClick?: () => void;
+  children: React.ReactNode;
+}
 
-const ReleaseDetailsPage = () => {
+const PopoverWrapper = ({ onClick, children }: PopoverWrapperProps) => {
+  return (
+    <PopoverButton
```

Comment:
I think PopoverWrapper is a bit confusing because it is not wrapping a popover. Wdyt about changing the name of your current PopoverButton to StyledFlex and then changing PopoverWrapper to PopoverButton?